opc/config.go

```diff
@@ -58,7 +60,24 @@ func (c *Config) Client() (*compute.ComputeClient, error) {
 
 	config.HTTPClient = httpClient
 
-	return compute.NewComputeClient(&config)
+	computeClient, err := compute.NewComputeClient(&config)
+	if err != nil {
+		return nil, err
+	}
+
+	opcClient := &OPCClient{
+		computeClient: computeClient,
+	}
+
+	if c.Storage {
```

Comment:
Would much rather have this without the extra boolean configuration, but I guess it's necessary?